### ISSUE TITLE
Which datasets to include in the Peekbank sample

### ISSUE THREAD

**Issue by mzettersten** [langcog/peekbank-paper#8]
One decision we need to make is what datasets will count towards the "final" sample for the BRM paper. Currently we are still "even" with the database version used in CogSci, but there are 3-5 datasets that are close or very close to being ready to be re-ingested.

I think my personal feeling would be that it might nice to use the eventual "version 2021.2" database with maybe 20-25 datasets for the BRM paper (so updated with some newly ingested datasets). Does that seem right?

**Comment by kachergis** [comment]
I agree that having somewhat more than the CogSci paper (20-25) would be great! feel free to ping me if there are any issues with importing the final few (I think I wrote import scripts for a couple eyecoder datasets from Virginia that haven't been added to the DB..)

**Comment by kachergis** [comment]
@JMankewitz did it look like there were any issues with the recent imports, or did you look at them?
